supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -410,6 +410,16 @@ NOTE: When referring generally to other Kubernetes components, such as pods, nod
 
 *See also*:
 
+[[opt-in]]
+==== image:images/yes.png[yes] Opt in (noun)
```

Comment:
Should this be lowercase "opt in" here and in the description? It doesn't need to be capitalized, correct? (Everywhere else that is referenced is lower)

Also, should be (verb) here, not (noun) (per the description)

Reply:
Well spotted. Updated.